api/handlers_movers.go

```diff
@@ -293,3 +293,97 @@ func (s *server) RunShowHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	w.Write(j)
 }
+
+func (s *server) StartTaskHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
✂️ Let's remove this handler

Reply:
I agree this is the code repetition, but it follows the principle of SINGLE responsibility. So, lets keep it this way, in future, it allows us if we add a separate start & stop route.